app/Models/Group.php

```diff
@@ -47,4 +47,14 @@ class Group extends Model
      * @var array
      */
     public static $sortable = ['name'];
+
+    /**
+     * Get the Group Type model associated with the group_type_id attribute.
+     *
+     * @return Rogue\Models\GroupType
+     */
+    public function getGroupTypeAttribute()
+    {
+        return GroupType::find($this->group_type_id);
```

Comment:
I don't think this makes all that much difference, but this made me think of Rails Active Record relationships where there are convenience methods for relationships, [it looks](https://laravel.com/docs/5.5/eloquent-relationships#one-to-one) like we could do something similar here with a `groupType` method and a `hasOne(GroupType)`.

EDIT: Looks like we use this on [Posts#review](https://github.com/DoSomething/rogue/blob/7b16ef1157d1442b944dcd7a020dc5b3bdb36b14/app/Models/Post.php#L155-L161)!

Reply:
I did try to use [`hasOne`](https://laravel.com/docs/5.5/eloquent-relationships#one-to-one) in a `group()` method per the Laravel docs, but it'd result in:

```
ErrorException: Undefined property: Illuminate\Database\Eloquent\Relations\HasOne::$name
```
when trying to access the name property of the return.

Looking more closely at the docs, I think this doesn't work because even though we do define a foreign key on Signups and Posts for the `group_id` fields -- there aren't any fields on the `Group` model that reference the `Signup` or `Post` models.

Reply:
Ahhhh right -- looks like in this case it's a `belongsTo` relationship! I wonder if that method would work!

Reply:
That still results in a similar error:
```
ErrorException: Undefined property: Illuminate\Database\Eloquent\Relations\BelongsTo::$name
```
also tried to specify the foreign/locak keys within the method and still get the error.

It sure feels like I'm doing something wrong though, because I'd expect this to work. @DFurnes -- anything you can 👀 as far as any 🐛 in getting these Eloquent relationships working here? I thought maybe it was because they'd only be used for query building, but the docs sure make it seem like this should work.

Reply:
> It sure feels like I'm doing something wrong though

I was! I didn't realize the relationship methods had to be added on each side of the relationship 🤦 . Fixed this up in https://github.com/DoSomething/rogue/pull/1066/commits/8aa8d792cd47aefbdf86c2853a354b185e4285e8, and going to make similar changes to the `Signup` and `Post` models (that's what I thought you were referring to at first when I first read this comment.. which was way too early to be reading 🌅 ☕ )

Reply:
🙏  #childNumberOneAndChildNumberTwo